### PR TITLE
DT-932 make Holocene component labels mandatory

### PR DESCRIPTION
**Pull request by rossedfort**
## Description & motivation 💭 
Make `label` props required across all Holocene components that can render a `<label />`. Also add a `labelHidden` prop that defaults to false. If true, add the `sr-only` class to the label

Components Affected
- Checkbox
- ChipInput
- Input
- NumberInput
- RangeInput
- Select
### Screenshots (if applicable) 📸 
n/a
### Design Considerations 🎨 
n/a
### How was this tested 👻 

- [x] Manual testing
- [ ] E2E tests added
- [ ] Unit tests added

### CODE REVIEW

File: src/lib/components/workflow/filter-search/list-filter.svelte [hunk 1]
```diff
@@ -22,10 +22,12 @@
 
 <div class="flex">
   <ChipInput
+    label={translate('type-or-paste-in')}
```
Comment (laurakwhit) [inline src/lib/components/workflow/filter-search/list-filter.svelte:25]: Would it be helpful to include the attribute here as well?

```suggestion
    label="{translate('type-or-paste-in')} {$filter.attribute}"
```

or even just the attribute

```suggestion
    label="{$filter.attribute}"
```

File: src/lib/components/workflow/filter-search/text-filter.svelte [hunk 2]
```diff
@@ -22,6 +22,8 @@
 </script>
 
 <Input
+  label={translate('search')}
```
Comment (laurakwhit) [inline src/lib/components/workflow/filter-search/text-filter.svelte:25]: Similarly, how about using the attribute here so it reflects what should go in the input?

```suggestion
  label={$filter.attribute)}
```

**Review by laurakwhit — APPROVED** [review]

File: src/lib/components/workflow/filter-search/list-filter.svelte (hunk 1 above)
Comment (rossedfort) [reply src/lib/components/workflow/filter-search/list-filter.svelte:25]: I like just using the attribute, good call 👍

File: src/lib/components/workflow/filter-search/text-filter.svelte (hunk 2 above)
Comment (rossedfort) [reply src/lib/components/workflow/filter-search/text-filter.svelte:25]: Will use the attribute same as list-filter

**Pull request merged**